kansha/app/comp.py

```diff
@@ -244,6 +245,7 @@ def set_config(self, config_filename, conf, error):
 
         # search_engine engine configuration
         self.search_engine = SearchEngine(**conf['search'])
+        fts_schema.Card.update_schema(self.card_extensions)
```

Comment:
> Le 12 janv. 2016 à 16:49, Romuald Texier-Marcadé <notifications@github.com> a écrit :
> 
> In kansha/app/comp.py:
> 
> > @@ -244,6 +245,7 @@ def set_config(self, config_filename, conf, error):
> >  
> >          # search_engine engine configuration
> >          self.search_engine = SearchEngine(**conf['search'])
> > +        fts_schema.Card.update_schema(self.card_extensions)
> 
> Imperative schema contruction?

Right. Imperative and « stateless » schema construction. Which is already what
you do thru the several `get_schema_def()` methods.

> 
> —
> Reply to this email directly or view it on GitHub.
>